foundation/foundation-configuration-store/src/main/java/com/boozallen/aissemble/configuration/ConfigStoreInit.java

```diff
@@ -20,39 +23,82 @@
 import javax.enterprise.context.ApplicationScoped;
 import javax.enterprise.inject.Any;
 import javax.enterprise.inject.spi.CDI;
-import java.util.HashSet;
+
+import java.util.HashMap;
+import java.util.Map;
 import java.util.Set;
 
+import org.apache.commons.lang3.StringUtils;
+
 /**
  * ConfigStoreInit loads configuration properties as application starts up
  */
 @Startup
 @ApplicationScoped
 public class ConfigStoreInit {
     private static final Logger logger = LoggerFactory.getLogger(ConfigStoreInit.class);
+    private static Map<String, String> status = new HashMap<>();
```

Comment:
Q: Can we just use String for the status since the status can only be "loading completed" or "Skip Load"? If we can, probably makes sense to create a enum for the status